integration/airflow/scripts/run-dev-airflow.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+#
+# Copyright 2018-2022 contributors to the OpenLineage project
+# SPDX-License-Identifier: Apache-2.0
+#
+# Usage: $ ./run-dev-airflow.sh
+
+set -e
+
+# Change working directory to integration
+project_root=$(git rev-parse --show-toplevel)
+current_dir=$(pwd)
+
+REBUILD="false"
+while [[ $# -gt 0 ]]
+do
+  case $1 in
+    -r|--rebuild)
+       shift
+       REBUILD="true"
+       ;;
+     *) echo "Unknown argument: $1"
+      shift
+      ;;
+  esac
+done
+
+
+if [[ "${REBUILD}" == "true" ]]; then
+   echo "Rebuilding..."
+   cd "${project_root}"/integration
+   cp -r ../client/python python
+   docker build -f airflow/Dockerfile.tests -t openlineage-airflow-base .
+   rm -rf python
+   cd ${current_dir}
+fi
+
+# this makes it easier to keep proper permissions on logs
+mkdir -p airflow/logs
+chmod a+rwx -R airflow/logs
+
+# maybe overkill
+OPENLINEAGE_AIRFLOW_WHL=$(docker run --rm openlineage-airflow-base:latest sh -c "ls /whl/openlineage*.whl")
+OPENLINEAGE_AIRFLOW_WHL_ALL=$(docker run --rm openlineage-airflow-base:latest sh -c "ls /whl/*")
+
+# Add revision to requirements.txt
+cat > requirements.txt <<EOL
```

Comment:
The less extra requirements to run tests, the better. 
Please consider, perhaps in an extra PR, removing those requirements and make use of a list available in `setup.py`.